pyannote/audio/core/io.py

```diff
@@ -75,20 +83,35 @@ class Audio:
     >>> audio = Audio(sample_rate=16000, mono=True)
     >>> waveform, sample_rate = audio({"audio": "/path/to/audio.wav"})
     >>> assert sample_rate == 16000
-
-    >>> two_seconds_stereo = np.random.rand(44100 * 2, 2, dtype=np.float32)
-    >>> waveform, sample_rate = audio({"waveform": two_seconds_stereo, "sample_rate": 44100})
+    >>> sample_rate = 44100
+    >>> two_seconds_stereo = torch.rand(2, 2 * sample_rate)
+    >>> waveform, sample_rate = audio({"waveform": two_seconds_stereo, "sample_rate": sample_rate})
     >>> assert sample_rate == 16000
-    >>> assert waveform.shape[1] == 1
+    >>> assert waveform.shape[0] == 1
     """
 
     @staticmethod
-    def normalize(waveform: np.ndarray) -> np.ndarray:
-        return waveform / (np.sqrt(np.mean(waveform ** 2)) + 1e-8)
+    def normalize(waveform: Tensor) -> Tensor:
+        """
+
+        Parameters
+        ----------
+        waveform : (channel, time) Tensor
+            Single or multichannel waveform
+
+
+        Returns
+        -------
+        waveform: (channel, time) Tensor
+        """
+
+        means = waveform.mean(dim=1)[:, None]
+        stds = waveform.std(dim=1)[:, None]
+        return (waveform - means) / (stds + 1e-8)
```

Comment:
Actually, this is not the same. `normalize` is not meant as `standardize` (mean substraction and putting variance to one).  

It is meant as normalizing the power of the signal so that we can later control the signal-to-noise ratio when summing two chunks.

Reply:
I see, I think torchaudio does that out of the box? https://github.com/pytorch/audio/blob/3b9e93372dd48649624ac2bbf660bb2e3384820e/torchaudio/backend/_soundfile_backend.py#L93